webpack/components/AnsibleRolesAndVariables/__test__/__snapshots__/AnsibleRolesAndVariablesImport.test.js.snap

```diff
@@ -74,6 +74,7 @@ exports[`ImportRolesAndVariablesTable should render 1`] = `
     expandId="expandable-toggle"
     gridBreakPoint="grid-md"
     isStickyHeader={false}
+    isTreeTable={false}
```

Comment:
Unrelated change but tests fail without it as the prop was added in a newer version of dependency.